src/toastAndroid.re

```diff
@@ -0,0 +1,22 @@
+type duration =
+  | BOTTOM
+  | CENTER
+  | LONG
+  | SHORT
+  | TOP;
+
+type gravity =
+  | BOTTOM
+  | CENTER
+  | TOP;
+
+[@bs.module "react-native"] [@bs.scope "ToastAndroid"]
+external show : (string, duration) => unit = "";
+
+[@bs.module "react-native"] [@bs.scope "ToastAndroid"]
+external showWithGravity : (string, duration, gravity) => unit = "";
+
+[@bs.module "react-native"] [@bs.scope "ToastAndroid"]
+external showWithGravityAndOffset :
+  (string, duration, gravity, int, int) => unit =
```

Comment:
do we really pass the variants as is to JS?

Reply:
Yeah, bindings were totally broken, fixed them already so you can check once again :)